src/app/containers/MediaPlayer/index.jsx

```diff
@@ -72,6 +72,11 @@ const MediaPlayerContainer = ({
       aresMediaBlock,
     ),
     type: format === 'audio' ? 'audio' : 'video',
+    guidanceMessage: pathOr(
```

Comment:
Ramda [`path`](https://ramdajs.com/docs/#path) might be more appropriate here. It returns `null` by default.

Reply:
Actually according to the docs, it returns undefined https://ramdajs.com/docs/#path, we need to use pathOR inorder to return null when the `warnings` object is empty.

Reply:
If `guidanceMessage` is `undefined` it will be set to `null` here: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-media-player/src/Placeholder/index.jsx#L81. It shouldn't require explicitly setting to `null`.